### PR TITLE
Feature/Image

### PR DESCRIPTION
**Pull request by michaelabudde**
## Describe your changes

Created Image Component with a default, hard coded image. The image src, alt and size can be changed depending on the screen or page section. 

## Issue ticket number and link

EES 12 https://tripleten-apiary.atlassian.net/browse/EES-12

Screenshot 2024-09-08 at 7 47 50 AM

## Checklist before requesting a review 

- [ ] I have performed a self-review of my code
- [ ] If there are changes to components, I have added / updated automated tests
- [ ] I have made any necessary updates to Storybook to accompany these changes
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by michaelabudde** [comment]
This branch is now ready for review, thank you for any feedback!

File: src/components/Form/Form.css
Comment (RPinkha) [inline src/components/Form/Form.css:1]: All these styles can be achieved via tailwind. Please look at the docs and apply to correct styles.

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import "./App.css";
+
+import { Image } from "../Image/Image";
+function App() {
+  // import closeModal from useModal if you want a button to also close the modal
+
+  return (
+    <div className="App">
```
Comment (RPinkha) [inline src/components/App/App.tsx:8]: Please use tailwind for the app component

File: src/components/CheckboxInput/CheckboxInput.css
Comment (RPinkha) [inline src/components/CheckboxInput/CheckboxInput.css:1]: Not sure if this was pulled from main, or if you coded this, but this doesn't belong on this branch.

**Review by RPinkha — COMMENTED** [review]
Good job. 
Need to migrate to tailwind as you used strictly BEM. 
Some components and files that don't belong are also present, but I think it has to do with the pull from a tainted main.

**Comment by michaelabudde** [comment]
All comments made here will be resolved in the new branch/pull request which is pulled from the updated Main.

**Pull request closed without merging**